src/ast/node/primary.rs

```diff
@@ -104,6 +104,7 @@ impl Node for NumNode {
 
 #[node]
 pub struct VarNode {
+    /// identifier name of a variable
```

Comment:
not only variable but also types. I think you could use `symbol` to refer to them